src/cdk/lib/stages/applications.ts

```diff
@@ -243,7 +243,6 @@ export class MicroservicesStack extends Stack {
                             AWS_REGION: Stack.of(this).region,
                             PETFOOD_FOODS_TABLE_NAME: imports.dynamodbExports.petFoodsTable.tableName,
                             PETFOOD_CARTS_TABLE_NAME: imports.dynamodbExports.petFoodsCartTable.tableName,
```

Comment:
We should start passing the name of the parameter to the applications. If you look at the `createOutput` method, the prefix for the parameter name can be modified (`PARAMETER_STORE_PREFIX`).

The additional benefit of doing it this way is that we can track dependencies better (we'll see in the environment variables which parameters are consumed by each microservice)

See an example here:

https://github.com/aws-samples/one-observability-demo/blob/962e576ed3c1c9305ef87f2fc9622e4477db76c9/src/cdk/lib/serverless/canaries/traffic-generator/traffic-generator.ts#L15

Reply:
what you say is that we shouldn't pass env vars but just use SSM params?

Reply:
I imagine you are using this parameter in the code?

                        imagescdnurl: `https://${this.distribution.distributionDomainName}/images`,

Instead of having the code looking for a parameter with this name hard coded we should pass the parameter name as an environment variable